samples/subsys/display/lvgl/boards/stm32h7b3i_dk.overlay

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2023, Fabian Blatz <fabianblatz@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
```

Comment:
drop one blank line, also the ones on line 9 and 15, keep it compact, same in the other overlay files